commons-statistics-descriptive/src/test/java/org/apache/commons/statistics/descriptive/TestHelper.java

```diff
@@ -30,6 +30,23 @@ final class TestHelper {
     /** Class contains only static methods. */
     private TestHelper() {}
 
+    /**
+     * Helper function to concatenate two arrays.
+     *
+     * @param array1 First array to be concatenated.
+     * @param array2 Second array to be concatenated.
+     * @return A new array containing elements from both input arrays in the order they appear.
+     */
+    static double[] concatenate(double[] array1, double[] array2) {
```

Comment:
Originally I thought this could be removed by changing your `computeExpected` to accept multiple arrays. But then it is simpler to compute the stat from a single array. This pattern is better going forward when a test may compute other more complicated stats such as the higher order moments.

However it may be helpful for this to be generic, i.e. allow more than 2 arrays. You could then add better test cases for combine with more than 2 arrays:
```java
        return Arrays.stream(arrays).flatMapToDouble(Arrays::stream).toArray();
```
It's not as efficient when you only have 2 arrays but it is more flexible.